docs/modules/configuring/con-configuring-vc-listeners.adoc

```diff
@@ -0,0 +1,276 @@
+[id='con-configuring-vc-listeners-{context}']
+= Configuring virtual cluster listeners
+
+[role="_abstract"]
+
+Clients make connections to virtual cluster listeners. Each listener defines a bootstrap address.  This is the address
+that the client makes its initial connection to. The listener also provides a means for the clients to talk to the
+proxied brokers. There is a choice about how this is done:
+
+* *Port Identifies Node* - the listener binds separate ports - one for each broker as well as a separate one for bootstrap.
+  Clients make connections to the different _port numbers_ as they need to converse with each broker.
+* *SNI Host Identifies Node* - the listener assigns a distinct hostname to each broker.
+  Clients make connections to the different _hostnames_ as they need to converse with each broker.
+  The listener uses SNI (Server Name Identification) to work out which broker the client wishes to talk to
+
+IMPORTANT: You must make sure that the listener's bootstrap address and generated broker addresses are resolvable
+           and routable by the Kafka Client.  You must also make sure firewall rules permit traffic to required port numbers.
+
+== Port Identifies Node
```

Comment:
I think we should note somewhere in this section that there is an implicit mapping between port and broker id and maybe that sparse ID ranges cause problems.

Reply:
if you are using ranges,we  don't actually have that problem, do we?

io.kroxylicious.proxy.internal.clusternetworkaddressconfigprovider.RangeAwarePortPerNodeClusterNetworkAddressConfigProviderTest#discoveryAddressMapMultipleRanges

Reply:
We sort of still do, in that we need a hint to people to define multiple ranges if they have disparate id groups or huge id numbers we need to know the base offset. 

Though at that point it might be fine in the details of the configuration.